packages/rocketchat-importer/server/classes/ImporterBase.coffee

```diff
@@ -181,6 +181,7 @@ Importer.Base = class Importer.Base
 							attachment.video_size = file.size
 
 						msg =
+							_id: details.slack_msg_id
```

Comment:
This needs to be more generic.  The importer package is a framework and so everything needs to be generic about it.